src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -821,6 +840,10 @@ WinnerElectionMode getWinnerElectionMode() {
     return mode == null ? WinnerElectionMode.MODE_UNKNOWN : mode;
   }
 
+  boolean isSingleWinnerEnabled() {
+    return getWinnerElectionMode() == WinnerElectionMode.STANDARD;
```

Comment:
Can we rename `STANDARD` to `STANDARD_SINGLE_WINNER` or something? Just calling it "standard" is too ambiguous.